### PR TITLE
Add turbo fix-lint command

### PR DESCRIPTION
**Pull request by ericanderson**

### CODE REVIEW

File: examples/basic/cli/package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
   "scripts": {
     "clean": "rm -rf lib dist types build tsconfig.tsbuildinfo",
     "dev:transpile": "tsup --watch",
+    "fix-lint": "eslint . --fix && dprint fmt --config $(find-up dprint.json)",
     "lint": "eslint . && dprint check  --config $(find-up dprint.json)",
     "prettier": "prettier .",
```
Comment (pashakondratyev) [inline examples/basic/cli/package.json:23]: Should we remove `prettier` as well?

Comment (ericanderson) [reply examples/basic/cli/package.json:23]: Good call.

File: examples/basic/cli/package.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
   "scripts": {
     "clean": "rm -rf lib dist types build tsconfig.tsbuildinfo",
     "dev:transpile": "tsup --watch",
+    "fix-lint": "eslint . --fix && dprint fmt --config $(find-up dprint.json)",
```
Comment (pashakondratyev) [inline examples/basic/cli/package.json:21]: Also is there a good convention for dashes vs colons? I.e. `fix:lint`

**Review by pashakondratyev — APPROVED** [review]

File: examples/basic/cli/package.json (hunk 2 above)
Comment (ericanderson) [reply examples/basic/cli/package.json:21]: Hmm. Not really. I'm going to leave this for now and we can come back to it later. Earlier versions of turbo didn't like the colon but its fine now obviously. 

In this case I used the colon as the dev:transpile is a "child" task of the virtual "dev" task.

Comment (ericanderson) [reply examples/basic/cli/package.json:21]: I guess we could say that typecheck is a child of `lint` and arguably probably should be. But typescript doesnt currently let us use build mode and `--noEmit` complicating things. So it doesnt just lint check, it has a side effect of creating `build/types/*`.

Do you have a strong opinion on any of this?

**Pull request merged**